modules/file-integrity-understanding-file-integrity-node-statuses-object.adoc

```diff
@@ -23,17 +22,37 @@ worker-fileintegrity-ip-10-0-165-160.ec2.internal   102s
 ----
 
 [NOTE]
-====
-`FileIntegrityNodeStatus` might not be created until the second run of the
-scanner is finished. The period is configurable.
+The `FileIntegrityNodeStatus` object might not be created until the second run of the scanner is finished. The period is configurable.
 ====
 
-There is one result object per node. The `nodeName` attribute of
-each `FileIntegrityNodeStatus` object corresponds to the node being scanned. The
-status of the file integrity scan is represented in the `results` array, which
-holds scan conditions.
+There is one result object per node. The `nodeName` attribute of each `FileIntegrityNodeStatus` object corresponds to the node being scanned. The
+status of the file integrity scan is represented in the `results` array, which holds scan conditions.
 
 [source,terminal]
 ----
 $ oc get fileintegritynodestatuses.fileintegrity.openshift.io -ojsonpath='{.items[*].results}' | jq
 ----
+
+The `fileintegritynodestatus` object reports the latest status of an AIDE run and exposes the status as `Failed` or `Succeeded` in a `status` field.
```

Comment:
This is not accurate. There is one more status called `Errored`.
Should be:
The `fileintegritynodestatus` object reports the latest status of an AIDE run and exposes the status as `Failed` or `Succeeded` or `Errored` in a `status` field.